src/test/ddms_tests.cpp

```diff
@@ -186,8 +199,7 @@ BOOST_AUTO_TEST_CASE(shouldOnlyModifyLicenseIfAlreadyPushed)
 	auto coinbaseTx = createCoinbase();
 	auto lTx = createLicenseTransaction(coinbaseTx.GetHash());
 
-	unsigned char rawAddress[] = {169, 20, 230, 53, 247, 106, 15, 189, 177, 112, 86, 88, 161, 227, 176, 86, 115, 120, 7, 74};
-	std::string address(rawAddress, rawAddress + 20);
+	std::string address{"a914e635f76a0fbdb1705658a1e3b0567378074a"};
```

Comment:
This is a script, not address. As a side note, P2SH scripts should end with `87` byte

Reply:
Done